src/plugins/deploy/azureDeployPlugin.ts

```diff
@@ -21,17 +21,27 @@ export class AzureDeployPlugin {
               "list"
             ]
           }
+        },
+        options: {
+          "resourceGroup": {
+            usage: "resource group for the service",
+            shortcut: "g",
+          }
         }
       }
     }
   }
 
+  private log(message: string) {
```

Comment:
Instead of introducing this here consider creating some base `ServerlessPlugin` class that all our plugins inherit from to provide some base functionality like this `log(...)` function..

Reply:
good idea.  I'm going to undo this and do the changes in a separate pr